### ISSUE TITLE
data source

### ISSUE THREAD

**Issue by Mimetis** [arkiaconsulting/terraform-provider-akc#1]
Do you plan to add `Data Sources` as well ?

Something like

```
data "azurerm_app_configuration" "test" {
  name                = "existing"
  resource_group_name = "existing"
}

data "akc_key_value" "config_value" {
  endpoint = azurerm_app_configuration.test.endpoint
  key      = "Key"
}

output "my_output_value" {
  value = data.akc_key_value.config_value.value
}

```

**Comment by arkiaman** [comment]
Absolutely, let me think about it.

Could you please share your use case ?

**Comment by Mimetis** [comment]
Hey @arkiaconsulting 
The user story is quite simple.

I have on terraform project, used to create a bunch of things in my Azure environment.
This project will contains the **Azure Configuration** creation AND the keys-values needed.

Then I have another terraform project, that needs to retrieve "things".
These "things" are stored in the **Azure Configuration** app.

In a nutshell, one tf to create / write, and another to read

**Comment by arkiaman** [comment]
Thanks @Mimetis

**Comment by arkiaman** [comment]
Closed by #2

Version 0.2.0 is out

```terraform
data "akc_key_value" "my_value" {
  endpoint  = azurerm_app_configuration.test.endpoint
  label     = "Dev"
  key       = "Key"
}
```
=> data.akc_key_value.my_value.value
```terraform
data "akc_key_secret" "my_secret_id" {
  endpoint  = azurerm_app_configuration.test.endpoint
  label     = "Dev"
  key       = "Key"
}
```
=> data.akc_key_secret.my_secret_id.secret_id

**Comment by Mimetis** [comment]
Thanks @arkiaconsulting !
Will test it asap
